### PR TITLE
Add NFC guide to learn section

### PR DESCRIPTION
**Pull request by YamenMerhi**
This guide is like a starting point for the people looking at information regarding LUKSO and NFC.
It doesn't have any special content but could be a good start and be later improved with a full guide, now it just highlight the key point that people should consider.

### CODE REVIEW

**Review by Hugoo — COMMENTED** [review]
I think you could add a topic about types of NFC chips.

Some are just storing data and do "open URL" and some are smarter and can actually SIGN messages. The later offers much higher features and security.

As using NFC just to open a link is imo worthless.

There are also great info in here for instance


https://developer.apple.com/design/human-interface-guidelines/nfc

Might be worth adding in the doc

And maybe this too

https://iyk.app/

File: docs/learn/expert-guides/NFC-NFT.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+LUKSO's ecosystem utilizes specific token standards, mainly **[LSP7-DigitalAsset](../../standards/tokens/LSP7-Digital-Asset.md)** and **[LSP8-IdentifiableDigitalAsset](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md)**, which offer significant improvements over the traditional [ERC721] standard. These standards provide enhanced features such as richer metadata and more robust asset interactions. When developing NFC-based solutions for LUKSO, aligning with these standards ensures a better user experience.
+
+## Conclusion
+
+The NFC designed to be working on any EVM network will work on LUKSO but won't be compatible with its ecosystem, because the LUKSO ecosystem uses [different tools and standards](../migrate-to-lukso.md) than the one available on Ethereum. The only difference that needs to be made is to use the new token standards instead of [ERC721] or [ERC1155].
+
+### Further Information
+
+- [Phygital Hackathon Project Video](https://www.youtube.com/watch?v=NZiShK34YZ8).
+- [Phygital Hackathon Project Github](https://github.com/Tuszy/phygital).
```
Comment (Hugoo) [inline docs/learn/expert-guides/NFC-NFT.md:36]:
```suggestion
- [Phygital Hackathon Project GitHub](https://github.com/Tuszy/phygital).
```

**Comment by Hugoo** [comment]
@YamenMerhi can we merge? What are we missing besides the comment above?

**Comment by YamenMerhi** [comment]
@Hugoo I'll take a look at your comments above and modify the document

**Pull request merged**